.gitignore

```diff
@@ -1,2 +1,2 @@
 ### Composer template
-/vendor/
+/vendor/
```

Comment:
Good practice is enter a line break at the end of a file, as the original one

Reply:
added line break